cmd/vulcan-lucky/main.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2019 Adevinta
+Copyright 2022 Adevinta
```

Comment:
I think this is not required. We would put the 2022 header just if the file has been created on 2022. But I might be wrong.

Reply:
Addressed in 83d2167c389d0f4be6f076ccd271a29ad32e1c77